crates/client/rpc/src/lib.rs

```diff
@@ -608,7 +608,9 @@ where
             BroadcastedTransaction::DeployAccount(deploy_tx) => !deploy_tx.is_query,
         });
         if is_invalid_query_transaction {
```

Comment:
reverse the bool:
- remove all the `!` in the match
- rename `is_invalid_query_transaction` to `is_query`
- change `if is_invalid_query_transaction` to `if !is_query {}`
-  call `.estimate_fee(substrate_block_hash, tx, is_query)`

It's better than taking the inverse of the inverse

Reply:
That's nicer, thanks!